src/styles/blocks/section2.scss

```diff
@@ -0,0 +1,67 @@
+.section_2 {
+  margin-top: 79px;
+  text-align: justify;
+  width: 166px;
+  height: 81px;
+}
+
+.section_2__stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 5px;
+}
+
+.section_2__stars_star {
+  background-image: url(../images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  margin: 0;
+  background-position: center;
+}
+
+.section_2__stars_star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
```

Comment:
It is recommended to use SCSS variables for image paths to ensure easier maintenance and to avoid duplication. Consider defining a variable for the star images.